packages/ra-ui-materialui/src/button/CreateButton.tsx

```diff
@@ -16,6 +16,7 @@ const CreateButton: FC<CreateButtonProps> = props => {
         classes: classesOverride,
         label = 'ra.action.create',
         icon = defaultIcon,
+        variant = 'text',
```

Comment:
Why do you add a default value? that's a breaking change. Your code should work without a default value.

Reply:
You're right - I've removed default.